scipy/special/__init__.py

```diff
@@ -664,8 +664,10 @@
     spherical_in,
     spherical_kn
 )
+from ._boost import _nbinom_cdf as nbdtrik
 
 __all__ = _ufuncs.__all__ + _basic.__all__ + orthogonal.__all__ + [
+    'nbdtrik',
```

Comment:
I put `nbdtrik` here for testing but we might need to change it place to be cleaner.